js/main.js

```diff
@@ -1,2 +1,129 @@
 $(function () {
+
+  let humanChoice = ""
+  let computerChoice = ""
+
+  let humanSelection = 0
+  let computerSelection = 0
+
+  let humanScore = 0
+  let computerScore = 0
+
+
+
+  function statusMessage() {
+    $("#status").html("You played <u>" + humanChoice +  "</u>. The computer played <u>" + computerChoice + "</u>.")
+  }
+
+  function getComputerSelection() {
+    if( computerSelection === 1 ){
+      computerChoice = "Rock"
+    } else if ( computerSelection === 2 ) {
+      computerChoice = "Paper"
+    } else{
+      computerChoice = "Scissors"
+    }
+
+  }
+
+
+  function compare(){
+
+    getComputerSelection()
+
+    if (( humanSelection === 1 && computerSelection === 3 ) || ( humanSelection === 2 && computerSelection === 1 ) || ( humanSelection === 3 && computerSelection === 2 )){
+
+      humanScore++
+
+      $("#computerScore").html(computerScore)
+      $("#humanScore").html(humanScore)
+      statusMessage()
+      $("#winner").html("You win!")
```

Comment:
@jl1982 Great job on separating code blocks into individual functions. To that note, this part could have also been expressed in a function with the winner message, `win` or `loose`, being passed in as an argument. 

check out the solution for an [alternative](https://github.com/jsr20190826/rock-paper-scissors-solution/blob/1583be48146e2521b25308e017c52fbfcf0d9d89/js/main.js#L74) approach